src/natcap/invest/urban_nature_access.py

```diff
@@ -0,0 +1,315 @@
+import os
+import logging
+import shutil
+import tempfile
+
+import pygeoprocessing
+import taskgraph
+from osgeo import gdal
+from osgeo import osr
+import numpy
```

Comment:
I think Dave has convinced me that alphabetically ordering the imports within their sections is a nice thing to do. I thought it might be a little early for such a small suggestion, but the more of these small things that get worked out the less there will be later!

Reply:
Patched in 442e84e9 using [`isort`](https://pycqa.github.io/isort/index.html) and some configuration for it in `pyproject.toml`.